nmmo/core/map.py

```diff
@@ -64,7 +64,7 @@ def reset(self, map_id):
 
   def step(self):
     '''Evaluate updatable tiles'''
-    self.realm.log_milestone('Resource_Depleted', len(self.update_list),
+    self.realm.log_milestone('[MAP] Resource_Depleted', len(self.update_list),
```

Comment:
Do we want [MAP] here? I assume the milestone name should be a string with no spaces or []

Reply:
done!